VoTCore/Exeptions/EntryAlreadyExistsExeption.cs

```diff
@@ -8,7 +8,7 @@
 namespace VoTCore.Exeptions
 {
     /// <summary>
-    /// Exception for entry witch already exists in a Dictonary
+    /// Exception for entry which already exists in a Dictonary
```

Comment:
"that" statt "which"
sonst ist es nicht auf entry bezogen. Bedeutet dann, dass die Exception im dictionary existiert und nicht der Eintrag(entry) selbst

Reply:
done